.github/workflows/bootstrap.yml

```
@@ -20,14 +20,12 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'yarn'
 
-      - name: Cache
+      - name: Cache node_modules
         uses: actions/cache@v3
```

actionlint (1.7.9)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)